GVFS/GVFS.Common/ProcessHelperImpl.cs

```diff
@@ -0,0 +1,16 @@
+namespace GVFS.Common
+{
+    /// <summary>
+    /// Default product implementation of IProcessHelper
+    /// interface. Delegates calls to static ProcessHelper class. This
+    /// class can be used to enable testing of components that call
+    /// into the ProcessHelper functionality.
+    /// </summary>
+    public class ProcessHelperImpl : IProcessHelper

```

Comment:
How much effort is it to also remove the `static class ProcessHelper` fully?

Reply:
We have ~100 references to `ProcessHelper`, not impossible, but still a bit of a change.

Reply:
Noted for followup